### ISSUE TITLE
Transferred comics don't appear in iPad

### ISSUE THREAD

**Issue by EdmundBenedict** [swisspol/ComicFlow#134]
I have transferred over a large  number of comics using the webserver but most of them don't appear when I'm looking through the library on the iPad, half of the folders I made don't even show up. There was no error message, the files are all .cbr or .cbz and I can still view and download the files when I use the webserver to look through them, but I still can't view them in the app, any idea what's wrong? The ipad is a new ipad pro.
Thanks

**Comment by swisspol** [comment]
What about the ComicFlow log? You can access it from the Settings pane inside the app. It will report any internal error e.g. if it fails to understand a given comic file.

**Comment by EdmundBenedict** [comment]
No, there were no errors reported for those transfers, and there was some errors reported from before I bought the unlimited webserver, so the log seems to work fine.

**Comment by swisspol** [comment]
Then the only explanation I can think of that the upload for the missing comic files didn’t complete. Try using another method to upload,

**Comment by spikebyt** [comment]
I'm a new user here and purchased the full server version  of ComicFlow to make reading from NAS possible. But I cannot find a way how to connect to my NAS and I would like to get some help wwith this. found nothing while searching so far

**Comment by swisspol** [comment]
Reading directly comics from a server or NAS is not something supported by ComicFlow.
